packages/dds/sequence/package.json

```diff
@@ -109,8 +109,8 @@
 		"test:memory": "mocha --config src/test/memory/.mocharc.cjs",
 		"test:memory-profiling:report": "mocha --config src/test/memory/.mocharc.cjs",
 		"test:mocha": "npm run test:mocha:esm && echo skipping cjs to avoid overhead - npm run test:mocha:cjs",
```

Comment:
path qualifications are also missing for these.

Reply:
Er. No. These don't require paths here, but either they should be here or the spec in the mocha config should be updated to ESM.